File: hw/cañarte/u3/HW29Composite&TemplateMethod/CompositePattern/src/main/java/ec/edu/espe/compositepattern/controller/Teller.java
package ec.edu.espe.compositepattern.controller;

/**
 *
 * @author Saray Cañarte
 */
public class Teller extends Employee {
    public Teller(String aName) {
        this();
        name = aName;
    }

    public Teller() {
        super();
        title = "Teller";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}

File: hw/Rodriguez/u3/hw29/hw29Python/Composite/controller/Teller.py
import sys
import os


sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Employee import Employee


class Teller(Employee):
    def __init__(self, name):
        super().__init__(name, "Teller")

    def add(self, employee):
        pass  

    def remove(self, employee):
        pass  